srml/support/src/storage/storage_vec_hashed.rs

```diff
@@ -0,0 +1,150 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
```

Comment:
Can someone give a brief explanation of how we eneded up having two versions of these traits (hashed, unhashed)? I assume that it has a particular reason/bottleneck that we have this code duplication and I can't see the big picture now.

Reply:
I think they are here for historical reason